src/Commands/Component/Illuminate/Filesystem/Installer.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * This file is part of Zero Framework.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace LaravelZero\Framework\Commands\Component\Illuminate\Filesystem;
+
+use LaravelZero\Framework\Commands\Component\Installer as InstallCommand;
+use LaravelZero\Framework\Contracts\Providers\Composer as ComposerContract;
+use LaravelZero\Framework\Contracts\Commands\Component\Installer as InstallerContract;
+
+/**
+ * This is the Zero Framework illuminate/filesystem install class.
+ *
+ * @author Nuno Maduro <enunomaduro@gmail.com>
+ */
+class Installer implements InstallerContract
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function install(InstallCommand $command, ComposerContract $composer): bool
+    {
+        $command->info('Pulling illuminate/filesystem...');
+        $composer->require('illuminate/filesystem "5.5.*"');
+
+        $command->info('Creating dummy assets directory...');
+        shell_exec('cd '.BASE_PATH.'&& mkdir assets');
```

Comment:
Should we call this folder `storage` instead of `assets`?

Reply:
For the folder, the name `assets` makes sense as it stores the "assets" But for the code `Storage` sounds better so kept it that way :smile: What do you say? Will change if you say